pkg/state/actors.go

```diff
@@ -1,51 +1,17 @@
 package state
 
-import (
-	"github.com/ipfs/go-cid"
-	dag "github.com/ipfs/go-merkledag"
+type ActorCodeCid int
+const (
+	AccountActorCodeCid  = ActorCodeCid(iota)
+	StorageMinerCodeCid
+	MultisigActorCodeCid
+	PaymentChannelActorCodeCid
 )
 
-// Builtin actor code CIDs.
-var (
-	InitActorCodeCid          cid.Cid
-	AccountActorCodeCid       cid.Cid
-	StorageMarketActorCodeCid cid.Cid
+type SingletonActorAddress int
```

Comment:
Structurally fine, but it's a bit confusing to use "cid" and "address" for things that aren't. Rename these types to `ActorCodeId` and `SingletonActorId`.